code/headers/module_task.hpp

```diff
@@ -27,7 +25,10 @@ namespace r2d2::module_scheduler {
     public:
         template <typename... Args>
         explicit module_task_c(Args &&... args)
-            : module(comm, std::forward<Args>(args)...) {
+            : rtos::task<StackSize>("task_module"),
+              module_task_base_c(),
+              module(comm, std::forward<Args>(args)...),
+              allocator() {
             auto &flag = create_waitable<rtos::flag>("start_flag");
```

Comment:
You don't need to save the result of `create_waitable` here

Reply:
fixed in https://github.com/R2D2-2019/module_scheduler/pull/2/commits/0b6bb1cc78da3fb493f61df5159a6da6174e1d93